pom.xml

```diff
@@ -455,10 +455,10 @@
                                 </requireReleaseDeps>
                                 <requireUpperBoundDeps />
                                 <requireMavenVersion>
-                                    <version>3.5.4</version>
+                                    <version>3.8.1</version>
                                 </requireMavenVersion>
                                 <requireJavaVersion>
-                                    <version>[11.0.10,12.0)</version>
+                                    <version>[11.0.11,12.0)</version>
```

Comment:
We should  give the teams a heads up; just upgraded my Java -- I was on 1.11.10.

Reply:
Tagged them as reviews, should get a notification